lib/cdo/test_run_utils.rb

```diff
@@ -53,7 +53,7 @@ def self.run_blockly_core_tests
   def self.run_dashboard_tests
     Dir.chdir(dashboard_dir) do
       with_hipchat_logging('dashboard tests') do
-        RakeUtils.rake_stream_output 'test'
+        RakeUtils.system_stream_output "RAILS_ENV=#{rack_env}", "RACK_ENV=#{rack_env}", 'bundle', 'exec', 'rails', 'test'
```

Comment:
In Rails 5, the [test runner](http://guides.rubyonrails.org/testing.html#the-rails-test-runner) got a substantial update, and the command to run all tests changed from `rake test` to `rails test`.